drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/_advanced_directives.haml

```diff
@@ -0,0 +1,27 @@
+= render 'field_section', f: f, fields: @assessment.advanced_directive_responses
+
+- content_for :page_js do
+  :javascript
+    $(document).ready(function() {
+      $('.jHasAdvancedDirective').change(function() {
+        if(this.value == 'yes') {
+          $('.jYesDirective').slideDown();
+          $('.jNoDirective').slideUp();
+        }
+        if(this.value == 'no') {
+          $('.jNoDirective').slideDown();
+          $('.jYesDirective').slideUp();
+        }
+      });
+
+      $('.jHasAdvancedDirective').each(function() {
+        if(this.value == 'yes') {
+          $('.jNoDirective').hide();
+        } else if(this.value == 'no') {
+          $('.jYesDirective').hide();
+        } else {
+          $('.jYesDirective').hide();
+          $('.jNoDirective').hide();
```

Comment:
These are fine, but generally I find adding and removing classes to be a better approach.  So instead of `.hide()`, I'd use `.addClass('hide')` Are you missing the show cases?  Seems odd that you can only hide tings.